crates/common/tedge_config/src/tedge_config_location.rs

```diff
@@ -30,17 +30,12 @@ impl TEdgeConfigLocation {
 
     /// `tedge.toml` is located in `/etc/tedge`.
     pub fn from_default_system_location() -> Self {
-        Self::from_custom_root(Path::new(DEFAULT_ETC_PATH).join("tedge"))
+        Self::from_custom_root(Path::new(DEFAULT_ETC_PATH))
     }
 
     /// `tedge.toml` is located in `${etc_path}/tedge`.
-    pub fn from_custom_etc_location(custom_etc_path: impl AsRef<Path>) -> Self {
-        Self::from_custom_root(custom_etc_path.as_ref().join("tedge"))
-    }
-
-    /// `tedge.toml` is located in `${home_path}/.tedge`.
-    pub fn from_users_home_location(home_path: impl AsRef<Path>) -> Self {
-        Self::from_custom_root(home_path.as_ref().join(".tedge"))
+    pub fn from_custom_location(custom_etc_path: impl AsRef<Path>) -> Self {
```

Comment:
Isn't this just a duplicate of `TEdgeConfigLocation::from_custom_root()`? Why duplicate it?